operators/src/plot/box_plot.rs

```diff
@@ -38,10 +38,9 @@ pub struct BoxPlotParams {
     /// Name of the (numeric) attributes to compute the box plots on.
     #[serde(default)]
     pub column_names: Vec<String>,
-
-    /// For rasters, we have the option to include no-data values
-    #[serde(default)]
-    pub include_no_data: bool,
+    // For rasters, we have the option to include no-data values
+    // #[serde(default)]
+    // pub include_no_data: bool,
```

Comment:
This makes no sense now right? I f no data has no value, we can no longer plot it right? If so, you can remove this line too

Reply:
yes